First draft of TTW history page

aleesteele opened the pull request:
### Summary

Will close #1731 and #2031

### List of changes proposed in this PR (pull-request)
* Initial draft has been taken from onboarding documents for the Turing Way team: https://hackmd.io/pBysnLn2QxKMCHqPKvj9uA?edit

### What should a reviewer concentrate their feedback on?

What does this history exclude? Who does it exclude? How is it incomplete? 

Should this page be include as a separate sub page of the book?

This would also be a great place to collect stories & experiences from the past year years of the project - as a means of writing a collective history of the project:
- What have your experiences been during X year of TTW?

### Acknowledging contributors

<!-- Please select the correct box -->

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/alan-turing-institute/the-turing-way/blob/main/README.md#contributors) in the README file: <!-- replace this text with the GitHub IDs of any new contributors -->

da5nsy commented on book/website/history.md:
```diff
@@ -0,0 +1,19 @@
+
+With the addition of Dr. Malvika Sharan in late 2019, The Turing Way expanded into five guides. As a co-founder of Open Life Science (which uses many Mozilla OL frameworks), Malvika helped to incubate what are now key partnerships for The Turing Way. With other key collaborators, she facilitated the creation of 4 more guides (on Project Design, Collaboration, Community, and Research Ethics), alon ...
+
+# Maintenance & Sustainability (2022-Ongoing)
```
```suggestion
## Maintenance & Sustainability (2022-Ongoing)
```

da5nsy commented on book/website/history.md:
```diff
@@ -0,0 +1,19 @@
+## History of The Turing Way project
```
```suggestion
# History of The Turing Way project
```

aleesteele replied:
Edited - thanks @da5nsy!

aleesteele commented:
![Alt: A screenshot of a powerpoint slide with a timeline on it, marked along four periods: 2018, 2019, 2020, and 2021. Each date features different visuals and descriptions.
2018: Text says: "Proposal by Kirstie Whitaker approved.". Mozilla logo. The Alan Turing Institute logo. Scriberia illustration of Kirstie imagining project.
2019: Picture of smiling Kirstie and others in office,. Text reads: "Book launched with 10 members, 3 chapters, many workshops, 2 Book Dashes. Year ended with Several chapters by 67 contributors."
2020: Text reads: "Book Dash 2020, Project expanded to five guides, Community Handbook, social media, Translation started. 238 contributors."
2021: Screenshot of Turing Way fireside chat logo, titled "What Exactly is The Turing Way?" Text reads: "Project growth, 134+ chapters, 1600 Twitter followers, 3000+ monthly visitors, citation in peer reviewed articles, government reports and policies, new projects emerged as extension. 309 contributors." Logo of HiddenREF.](https://github.com/alan-turing-institute/the-turing-way/assets/18509789/c41539fb-8c3d-4d26-a274-6db85b907bec)
- Caption: A screenshot of a powerpoint slide with a timeline on it, marked along four periods: 2018, 2019, 2020, and 2021. Each date features different visuals and descriptions.
2018: Text says: "Proposal by Kirstie Whitaker approved.". Mozilla logo. The Alan Turing Institute logo. Scriberia illustration of Kirstie imagining project.
2019: Picture of smiling Kirstie and others in office,. Text reads: "Book launched with 10 members, 3 chapters, many workshops, 2 Book Dashes. Year ended with Several chapters by 67 contributors."
2020: Text reads: "Book Dash 2020, Project expanded to five guides, Community Handbook, social media, Translation started. 238 contributors."
2021: Screenshot of Turing Way fireside chat logo, titled "What Exactly is The Turing Way?" Text reads: "Project growth, 134+ chapters, 1600 Twitter followers, 3000+ monthly visitors, citation in peer reviewed articles, government reports and policies, new projects emerged as extension. 309 contributors." Logo of HiddenREF.
- Documented as a tweet: https://twitter.com/turingway/status/1542862053739749379
- Documented as a google slide:

da5nsy commented:
> Thanks for your feedback, @da5nsy! This is very much a work in progress, and the plan is to invite testimonies from every stage of the project - it's a work in progress! 😊

I'm drawn to the red cross of a CI failure like a moth to a flame... 🦋 🔥

aleesteele commented:
Adding notes here from recent report:
- From 2018-2020, the project's foundations were established as a "Guide for Reproducible Research", with initial chapters, conceptualisations, and open source processes documented. A github repository was made for the project, and internal processes decided in how acknowledge of authorship works within the project (i.e. installation of Welcome bot, All-Contributors bot) and Collaborative spaces to contribute were created (i.e. Collaboration Cafe, Book Dash). In-person workshops were given in person on tools aligned with the project: particulary Binder and JupterBook, and the project was presented in many conferences across the UK.
- From 2020-early 2022: This period corresponded with the COVID-19 pandemic lockdowns, which transitioned many processes of contribution, collaboration, and documentation into remote ones. In part due to this, The Turing Way was able to expand greatly, from one guide on Reproducibility into five different guides related to project design, communication, collaboration, and research ethics. A global (and multilingual) community began to engage with The Turing Way, and the process of translation/localisation began informally. A Community Handbook was also established to document best practices that could be used by other communities. The Fireside Chat series was established in late 2021, facilitating shared spaces with networks and allies from across the open science ecosystem. The project received a few awards & citation in London Mayor report, HiddenREF Awards, Goldacre Report. Established Book Dash Planning Committee.
- From early 2022 onwards: Increased visibility of both The Turing Way project (public-facing book), and communities doing other kinds of work within the project (i.e. translation & localistaion, infrastructure maintenance). Incubation of 5 working groups: Editors & Reviers, Trainers & Mentors, Accessibility, Environmental Data Science, Infrastructure Maintainers.  The Turing Way organisational team has expanded from 8 to 30 people over the past year. Practioners Hub has been established as of 2023, with addition of RPM (Research Project Manager). Project presented in in-person and remote contexts (50+) [[citation 1](https://docs.google.com/presentation/d/1ZAwCe9hZCG-sntZgIKhfT5G8P0JWFDCK6Di7dde5OII/edit#slide=id.g2085abe9303_2_195)]

aleesteele commented:
After a conversation with @da5nsy - I'm switching this to a real PR! Adding this very basic version to TTW, with the idea of expanding it into more collective storytelling in the coming months. Can't let perfect get in the way of the good 😉 

@malvikasharan @KirstieJane - I know there's some outdated language here around the core team, but are there any foundational edits or corrections you would make to this basic version?

aleesteele commented:
Adding breadcrumbs to #2080 - which includes a 'welcome' and 'history' page

da5nsy commented:
> After a conversation with @da5nsy - I'm switching this to a real PR! Adding this very basic version to TTW, with the idea of expanding it into more collective storytelling in the coming months. Can't let perfect get in the way of the good 😉

Woohoo! Well done 🎉

jcolomb commented:
very nice, interesting to know the project is that young.
It would be easier to review if each sentence would end with a line break.

It may be interesting to write a section about the transformation of the turing way project into the turing way community.

I am also wondering if there is a relation between the rise of this community and the concomitant slow death of the openscienceMOOC project (2019-2020).

probably something to link inside a organisation level readme #3282

jcolomb commented on book/website/history.md:
```diff
@@ -0,0 +1,40 @@
+The project was incubated during Kirstie's time as a [Mozilla Open Leader](https://mozilla.github.io/leadership-training/), and was funded by a grant within the Turing.
+
+Due to close relationships with the Jupyter Project, the book was written in JupyterBook.
+A github repository was made for the project, and internal processes decided in how acknowledge of authorship works within the project (i.e. installation of Welcome bot, All-Contributors bot) and collaborative spaces and events to contribute to the guides were created (i.e. Collaboration Cafe, Book Dash).
```
```suggestion
A github repository was made for the project, and internal processes decided in how acknowledge of authorship works within the project (for instance the Welcome and All-Contributors bots were installed) and collaborative spaces and events to contribute to the guides were created (Collaboration Cafe, and Book Dash were organised).
```

jcolomb commented on book/website/history.md:
```diff
@@ -0,0 +1,40 @@
+
+Due to close relationships with the Jupyter Project, the book was written in JupyterBook.
+A github repository was made for the project, and internal processes decided in how acknowledge of authorship works within the project (i.e. installation of Welcome bot, All-Contributors bot) and collaborative spaces and events to contribute to the guides were created (i.e. Collaboration Cafe, Book Dash).
+In-person workshops were given in person on tools aligned with the project: particulary Binder and JupterBook, and the project was presented in many conferences across the UK and internationally.
```
```suggestion
In-person workshops were given on tools aligned with the project (particulary Binder and JupterBook).
The project was presented in many conferences across the UK and internationally.
```

jcolomb commented on book/website/history.md:
```diff
@@ -0,0 +1,40 @@
+## Development and Growth (2019-2021)
+
+With the addition of Dr. Malvika Sharan in late 2019, The Turing Way expanded into five guides.
+As a co-founder of Open Life Science (which uses many Mozilla OL frameworks), Malvika helped to incubate what are now key partnerships for The Turing Way.
```
```suggestion
As a co-founder of Open Life Science (which uses many Mozilla open leadership frameworks), Malvika helped to incubate what are now key partnerships for The Turing Way.
```

aleesteele replied:
corrected for this

jcolomb commented on book/website/history.md:
```diff
@@ -0,0 +1,40 @@
+A Community Handbook was also established to document best practices that could be used by other communities.
+The Fireside Chat series was established in late 2021, facilitating shared spaces with networks and allies from across the open science ecosystem.
+The project received a few awards & citations in the 2021 London Mayor report, HiddenREF Awards, and Goldacre Report.
+The Book Dash Planning Committe was established to plan the community's flagship collaboration event.
```
when?

aleesteele replied:
in 2022 - adding this for clarity, good catch @jcolomb!

jcolomb commented on book/website/history.md:
```diff
@@ -0,0 +1,40 @@
+## Maintenance & Sustainability (2022-Ongoing)
+
+In 2022, Malvika and Kirstie became co-leads of *The Turing Way*, and welcomed Anne Lee Steele as the new community manager.
+As the number of contributors to the project numbers has grown to over 400 people, The Turing Way has also developed into a kind of ["community of communities"](https://ben.balter.com/2019/07/18/a-community-of-communities-oscon-2019/) with many ongoing projects developing alongside the expansion of the guides, demonstrating the need for internal systems that enable the sustainability and maintena ...
```
```suggestion
As the number of contributors to the project has grown to over 400 people, The Turing Way has also developed into a kind of ["community of communities"](https://ben.balter.com/2019/07/18/a-community-of-communities-oscon-2019/).
Many ongoing projects have been developing alongside the expansion of the guides, demonstrating the need for internal systems that enable the sustainability and maintenance of the project more broadly.
```

jcolomb reviewed (commented):
some wording and questions

malvikasharan commented on book/website/history.md:
```diff
@@ -0,0 +1,43 @@
+The project has its roots in the [Mozilla Open Leadership](https://mozilla.github.io/leadership-training/), a programme that led to Kirstie's Mozilla Fellowship in 2016, which influenced her as a researcher and strengthened her commitment to working open.
+
+*The Turing Way* has been funded by The Alan Turing Institute since the very beginning.
+Due to close relationships with the Jupyter Project, the book was written in JupyterBook.
```
That is not the entire truth.
The goal of The Turing Way is to provide a 21st-century book -- not static, and something that allows interactivity and creativity. JupyterBook at the time was the only option that allowed combining executable notebooks with flat MarkDown files.
JupyterBook has since inspired Quarto's development that offers this feature.

I suppose the other reason that you are highlighting ties to our commitment to build on existing open source technology and ensure that we have the opportunity to contribute back to the Jupyter system that is a critical open source infrastructure.

malvikasharan replied:
That's true. I think there might be an issue somewhere. @sgibson91 can probably share here as she was involved in the early days.

malvikasharan replied:
Ok, so speaking with Kirstie:
- 21st-century element was priority as well as ensuring upstream, contributions
- Bookdown was not executable - and Jupyter was offering that solution
- Also, we had a close connection with the Jupyter Book team which could offer advice and support

EstherPlomp commented on book/website/history.md:
```diff
@@ -0,0 +1,43 @@
+
+## Founding (2018 - 2019)
+
+*The Turing Way* is the flagship project of the Tools, Practices, and Systems programme.
```
I'm not sure if this is the best sentence to start things with: 
1) that wasn't the case in the beginning and 
2) what does this tell anyone outside of the Turing?

KirstieJane replied:
Yes - agree with @EstherPlomp - TPS came after The Turing Way!

EstherPlomp commented on book/website/history.md:
```diff
@@ -0,0 +1,43 @@
+
+*The Turing Way* has been funded by The Alan Turing Institute since the very beginning.
+Due to close relationships with the Jupyter Project, the book was written in JupyterBook.
+A GitHub repository was made for the project, and internal processes decided in how acknowledge of authorship works within the project (i.e. installation of Welcome bot, All-Contributors bot) and collaborative spaces and events to contribute to the guides were created (i.e. Collaboration Cafe, Book Dash).
```
I don't think we can use i.e. according to the style guide?

malvikasharan replied:
```suggestion
A GitHub repository was made for the project, and internal processes decided in how acknowledge of authorship works within the project (i.e. installation of Welcome bot, All-Contributors bot) and collaborative spaces and events to contribute to the guides were created (such as Collaboration Cafe, Book Dash).
```

EstherPlomp commented on book/website/history.md:
```diff
@@ -0,0 +1,43 @@
+
+## Development and Growth (2019-2021)
+
+With the addition of Dr. Malvika Sharan in late 2019, The Turing Way expanded into five guides.
```
There's a bit of a repeat with the expanding into the five guides with the next paragraph, and I think it can be more clearly explained that Malvika provided the capacity to do so and did not literally add the five guides herself - although almost of course :)

malvikasharan replied:
Are you saying Malvika did not write all the 100s of chapters? 😆

KirstieJane replied:
Yes - these two things are related but not causal - we got additional funding and hired @malvikasharan AND expanded because reproducibility on its own is not sufficient to address the major challenges (including incentive challenges!) faced by research data scientists.

aleesteele replied:
a

EstherPlomp commented on book/website/history.md:
```diff
@@ -0,0 +1,43 @@
+As the number of contributors to the project numbers has grown to over 400 people, The Turing Way has also developed into a kind of ["community of communities"](https://ben.balter.com/2019/07/18/a-community-of-communities-oscon-2019/) with many ongoing projects developing alongside the expansion of the guides, demonstrating the need for internal systems that enable the sustainability and maintena ...
+Initial governance work began in 2022, with hybrid hubs hosting events across multiple time zones and working groups developed to formalise localisation and translation, reviewing & editing, training and mentorship, accessibility, and infrastructure within the project.
+
+2023 saw the incubation of The Turing Way Practioner's Hub that aims to bring The Turing Way into the hands and ways of working of industry leaders, as well as a transition into a separate Github organisation for The Turing Way project.
```
Maybe rewrite the next two sentences with The Turing Way as the main objective instead of the year. Now it looks like we're passively guided by time instead of via active thought/actions!

EstherPlomp reviewed (commented):
I think the overview looks complete! It does feel a bit like just facts are listed, so it is not super exciting to read. Perhaps add some quotes or a bit more personal story to the facts?

KirstieJane commented on book/website/history.md:
```diff
@@ -0,0 +1,43 @@
+## Development and Growth (2019-2021)
+
+With the addition of Dr. Malvika Sharan in late 2019, The Turing Way expanded into five guides.
+As a co-founder of Open Life Science (which uses many Mozilla OL frameworks), Malvika helped to incubate what are now key partnerships for The Turing Way.
```
This really needs to be more nuanced about how Open Life Science AND The Turing Way are inspired by the Mozilla Open Leadership training programme.

Both @malvikasharan and I credit Mozilla Open Leadership with transformational mentorship that has led us to contribute to changing the world through open and inclusive science practices.

OLS and TTW had parallel development rather than one coming before the other... I don't think OLS needs to be specifically mentioned in the history of The Turing Way project page.

@malvikasharan - what do you think?

[EDIT: I'm in a call with Malvika now - she agrees 😅]

aleesteele replied:
Very true! I think this was something that was visible to me when I joined the team - but might be confusing for people who are just looking to learn about the project's history in a specific, contained way!

KirstieJane commented on book/website/history.md:
```diff
@@ -0,0 +1,43 @@
+## Founding (2018 - 2019)
+
+*The Turing Way* is the flagship project of the Tools, Practices, and Systems programme.
+Founded by Dr Kirstie Whitaker, the project began as a Book of Reproducible Research in 2019.
```
Technically the funding came through in 2018 and then we launched at the SSI Collaborations Workshop in 2019.

aleesteele replied:
Adding this in - thought maybe it should be moved further up?

KirstieJane reviewed (commented):
Capturing a few notes from the Turing Way core team co-working!

I'll add a full review soon!

aleesteele commented:
Folks (particularly @da5nsy @jcolomb @KirstieJane @malvikasharan who have commented here), I'm so sorry for my late response to this pull request!! I was on annual leave for the past few weeks, and am finally getting caught up on your thoughtful comments. Just wanted to flag that I am working on an updated draft throughout this week, and will add more responses to the comments here too! Thanks again. ❤️

aleesteele commented:
Alright! Finally finished another review of this draft, apologies folks that it's taken a bit longer than expected!
- Added a bit more context to the founding section about the context, funding, and use of Jupyter Notebooks - nuanced by Kirstie and Malvika's comments (thank you!)
- Also added in a bit more: particularly removing OLS, and adding in the statement (that we always use in presentations!) about how the guides expanded also because of the wider needs of research data scientists (used your wording in particular - Kirstie!)
- Added a bit more to 2022 as well

kallewesterling commented:
Just wanted to drop a comment here to say that this is SUCH an interesting and important page to have as part of The Turing Way. I'd love to see this merged soon so we can bask in all your glorious work over the past few years! 🎉

aleesteele commented:
Thank you so much @kallewesterling! Can't believe I forgot that - great catch! ❤️ 

I made a few changes to this pull request: pulling in some text from #2033 to create the forward section where this text can be embedded. I'm still facing build breaks though - no-bad-latin.py? Hmmm

@da5nsy - if you want to review these changes, perhaps we might even be able to get this merged by the end of Book Dash? 😄

Arielle-Bennett commented on book/website/foreward/history.md:
```diff
@@ -0,0 +1,57 @@
+The project has its roots in the [Mozilla Open Leadership](https://mozilla.github.io/leadership-training/), a programme that led to Kirstie's Mozilla Fellowship in 2016, which influenced her as a researcher and strengthened her commitment to working openly.
+
+*The Turing Way* has been funded by The Alan Turing Institute since the very beginning.
+Due to close relationships with the Jupyter Project, the book was written in JupyterBook.
```
Didn't Kirstie and Malvika explain previously that it wasn't just that there was a close relationship with JupyterBook but that the book was also open source, executable, and a prior relationship existed?

aleesteele replied:
Good point! It's true that this phrase emphasises the relationship above the toolkit - edited this to better reflect that. Thank goodness for first drafts!

Arielle-Bennett commented on book/website/foreward/history.md:
```diff
@@ -0,0 +1,57 @@
+
+*The Turing Way* has been funded by The Alan Turing Institute since the very beginning.
+Due to close relationships with the Jupyter Project, the book was written in JupyterBook.
+A GitHub repository was made for the project, and internal processes decided in how to acknowledge authorship works within the project (i.e. installation of Welcome bot, All-Contributors bot) and collaborative spaces and events to contribute to the guides were created (such as the Collaboration Cafe and Book Dash).
```
```suggestion
A GitHub repository was made for the project, and internal processes decided how to acknowledge authorship works within the project (i.e. installation of Welcome bot, All-Contributors bot) and collaborative spaces and events to contribute to the guides were created (such as the Collaboration Cafe and Book Dash).
```
We can point specifically to the Authorship chapter in the community handbook here - we should probably also be clearer on the "internal processes", was it Kirstie or the original project team who made the authorship decisions?

da5nsy replied:
Just to flag - the use of "i.e." here is causing CI to fail

aleesteele replied:
Edited to "for example"

Arielle-Bennett commented on book/website/foreward/history.md:
```diff
@@ -0,0 +1,57 @@
+Due to close relationships with the Jupyter Project, the book was written in JupyterBook.
+A GitHub repository was made for the project, and internal processes decided in how to acknowledge authorship works within the project (i.e. installation of Welcome bot, All-Contributors bot) and collaborative spaces and events to contribute to the guides were created (such as the Collaboration Cafe and Book Dash).
+In-person workshops were given on tools aligned with the project (particulary Binder and JupterBook).
+The project was presented in many conferences across the UK and internationally.
```
```suggestion
The project was presented in many conferences across the UK and internationally, as well as conferences offering the first spaces for in-person Book Dash events.
```

Arielle-Bennett commented on book/website/foreward/history.md:
```diff
@@ -0,0 +1,57 @@
+## Development and Growth (2019-2021)
+
+With the addition of Dr. Malvika Sharan in late 2019 as the community manager, *The Turing Way* continued to evolve and grow as it received more support and visibility.
+Malvika helped to incubate what are now key partnerships for ***The Turing Way*** within the wider 'open science' ecosystem, and helped to usher in the expansion of the project.
```
```suggestion
Malvika helped to incubate what are now key partnerships for _The Turing Way_ within the wider 'open science' ecosystem, and helped to usher in the expansion of the project.
```

Arielle-Bennett commented on book/website/foreward/history.md:
```diff
@@ -0,0 +1,57 @@
+
+With the addition of Dr. Malvika Sharan in late 2019 as the community manager, *The Turing Way* continued to evolve and grow as it received more support and visibility.
+Malvika helped to incubate what are now key partnerships for ***The Turing Way*** within the wider 'open science' ecosystem, and helped to usher in the expansion of the project.
+This period corresponded with the COVID-19 pandemic lockdowns, which transitioned many processes of contribution, collaboration, and documentation into remote ones.
```
It's not clear in this sentence if we are talking about the project specifically or practices more broadly when we talk about the transition - The Turing Way was already well set up to manage asynchronous and remote collaboration in 2020, which actually made pivoting online, including the Book Dashes, much easier

Arielle-Bennett commented on book/website/foreward/history.md:
```diff
@@ -0,0 +1,57 @@
+This period corresponded with the COVID-19 pandemic lockdowns, which transitioned many processes of contribution, collaboration, and documentation into remote ones.
+For many, it also became clear that reproducibility on its own is not sufficient to address the major challenges (including incentive challenges!) faced by research data scientists.
+
+In part due to all these factors, *The Turing Way* was able to expand greatly, from one guide on reproducibility into five different guides related to project design, communication, collaboration, and research ethics.
```
This was already planned in Feb 2020 so I would move this sentence before talking about COVID-19

Arielle-Bennett commented on book/website/foreward/history.md:
```diff
@@ -0,0 +1,57 @@
+In part due to all these factors, *The Turing Way* was able to expand greatly, from one guide on reproducibility into five different guides related to project design, communication, collaboration, and research ethics.
+
+A global (and multilingual) community began to engage with *The Turing Way*, and the process of translation/localisation began informally.
+A Community Handbook was also established to document best practices that could be used by other communities.
```
```suggestion
A [Community Handbook](https://the-turing-way.netlify.app/community-handbook/community-handbook) was also established to document best practices that could be used by other communities.
```

Arielle-Bennett commented on book/website/foreward/history.md:
```diff
@@ -0,0 +1,57 @@
+
+In part due to all these factors, *The Turing Way* was able to expand greatly, from one guide on reproducibility into five different guides related to project design, communication, collaboration, and research ethics.
+
+A global (and multilingual) community began to engage with *The Turing Way*, and the process of translation/localisation began informally.
```
```suggestion
A global (and multilingual) community began to engage with *The Turing Way*, and the process of [translation/localisation](https://the-turing-way.netlify.app/community-handbook/translation) began informally.
```

aleesteele replied:
Woohoo - yes! We can cite that chapter now.

Arielle-Bennett commented on book/website/foreward/history.md:
```diff
@@ -0,0 +1,57 @@
+
+A global (and multilingual) community began to engage with *The Turing Way*, and the process of translation/localisation began informally.
+A Community Handbook was also established to document best practices that could be used by other communities.
+The Fireside Chat series was established in late 2021, facilitating shared spaces with networks and allies from across the open science ecosystem.
```
```suggestion
The [Fireside Chat series](https://the-turing-way.netlify.app/community-handbook/fireside-chat) was established in late 2021, facilitating shared spaces with networks and allies from across the open science ecosystem.
```

Arielle-Bennett commented on book/website/foreward/history.md:
```diff
@@ -0,0 +1,57 @@
+A global (and multilingual) community began to engage with *The Turing Way*, and the process of translation/localisation began informally.
+A Community Handbook was also established to document best practices that could be used by other communities.
+The Fireside Chat series was established in late 2021, facilitating shared spaces with networks and allies from across the open science ecosystem.
+The project received a few awards & citations in the 2021 London Mayor report, HiddenREF Awards, and Goldacre Report.
```
Please can we be more specific here? You can use the reporting documents from this time to find the hyperlinks and additional details

Arielle-Bennett commented on book/website/foreward/history.md:
```diff
@@ -0,0 +1,57 @@
+
+## Maintenance & Sustainability (2022-Ongoing)
+
+In 2022, Malvika and Kirstie became co-leads of *The Turing Way*, and welcomed Anne Lee Steele as the new community manager.
```
```suggestion
In 2022, Malvika and Kirstie became co-leads of *The Turing Way*, and welcomed Anne Lee Steele as the new Research Community Manager.
```

Arielle-Bennett commented on book/website/foreward/history.md:
```diff
@@ -0,0 +1,57 @@
+
+In 2022, Malvika and Kirstie became co-leads of *The Turing Way*, and welcomed Anne Lee Steele as the new community manager.
+
+As the number of contributors to the project has grown to over 400 people, *The Turing Way* has also developed into a kind of ["community of communities"](https://ben.balter.com/2019/07/18/a-community-of-communities-oscon-2019/).
```
```suggestion
As the number of contributors to the project has grown to over 400 people, *The Turing Way* has also developed into a ["community of communities"](https://ben.balter.com/2019/07/18/a-community-of-communities-oscon-2019/).
```

Arielle-Bennett commented on book/website/foreward/history.md:
```diff
@@ -0,0 +1,57 @@
+In 2022, Malvika and Kirstie became co-leads of *The Turing Way*, and welcomed Anne Lee Steele as the new community manager.
+
+As the number of contributors to the project has grown to over 400 people, *The Turing Way* has also developed into a kind of ["community of communities"](https://ben.balter.com/2019/07/18/a-community-of-communities-oscon-2019/).
+Many ongoing projects have been developing alongside the expansion of the guides, demonstrating the need for internal systems that enable the sustainability and maintenance of the project more broadly.
```
I think it would be helpful to name some explicit examples here - from this sentence it's not clear what is meant by "many ongoing projects have been developing alongside the expansion of the guides" 

Do you mean that the project is providing the community infrastructure to support smaller scale, or offshooting collaborations? 

It's also not clear why this in itself demonstrates the need for internal systems around maintenance, I think those are two separate things so should be in two different sentences.

Arielle-Bennett commented on book/website/foreward/history.md:
```diff
@@ -0,0 +1,57 @@
+As the number of contributors to the project has grown to over 400 people, *The Turing Way* has also developed into a kind of ["community of communities"](https://ben.balter.com/2019/07/18/a-community-of-communities-oscon-2019/).
+Many ongoing projects have been developing alongside the expansion of the guides, demonstrating the need for internal systems that enable the sustainability and maintenance of the project more broadly.
+Many projects have grown that have adopted and used practices from *The Turing Way*.
+The core team of the project, which brought together Turing staff and volunteer leaders of the project was brought in in 2022.
```
```suggestion
The core team of the project, which brought together Turing staff and volunteer leaders of the project was established in 2022.
```

Arielle-Bennett commented on book/website/foreward/history.md:
```diff
@@ -0,0 +1,57 @@
+Many projects have grown that have adopted and used practices from *The Turing Way*.
+The core team of the project, which brought together Turing staff and volunteer leaders of the project was brought in in 2022.
+
+Initial governance work began in 2022 with hybrid hubs hosting events across multiple time zones and working groups developed to formalise localisation and translation, reviewing & editing, training and mentorship, accessibility, and infrastructure within the project.
```
```suggestion
Initial governance work began in 2022 with hybrid hubs hosting Book Dash events across multiple geographic locations and working groups developed to formalise localisation and translation, reviewing & editing, training and mentorship, accessibility, and infrastructure within the project.
```

Arielle-Bennett commented on book/website/foreward/history.md:
```diff
@@ -0,0 +1,57 @@
+The number of talks and workshops given by community members also expanded.
+
+In 2023, *The Turing Way* delivery team expanded, with the addition of Alexandra Araujo Alvarez as TTW's first Project Manager.
+The Practioner's Hub was incubated with 5 Experts-in-Residence (EiR) that aims to bring *The Turing Way* into the hands and ways of working of industry leaders. 
```
```suggestion
[The Practitioner's Hub](https://www.turing.ac.uk/turing-way-practitioners-hub) was piloted with the first cohort of 5 Experts-in-Residence (EiR).
It aims to build a shared understanding of open science, reproducibility, accessibility and research ethics to enhance quality, rigour and integrity in data science and AI, across academia, government, and industry, catalysed by _The Turing Way_. 
```

Arielle-Bennett commented on book/website/foreward/history.md:
```diff
@@ -0,0 +1,57 @@
+
+In 2023, *The Turing Way* delivery team expanded, with the addition of Alexandra Araujo Alvarez as TTW's first Project Manager.
+The Practioner's Hub was incubated with 5 Experts-in-Residence (EiR) that aims to bring *The Turing Way* into the hands and ways of working of industry leaders. 
+The open infrastructure of the project has also expanded, as *The Turing Way* moved to separate Github organisation.
```
```suggestion
The open infrastructure of the project has also expanded, as *The Turing Way* moved to separate [Github organisation](https://github.com/the-turing-way/the-turing-way).
```

da5nsy commented on book/website/_toc.yml:
```diff
@@ -12,6 +12,11 @@ format: jb-book
   - chapters:
+      # ===== Foreword Section ========================================
+      - file: foreword/foreword
+        sections: 
```
```suggestion
        sections:
```

da5nsy commented:
The directory introduced here seems to be misspelled ("foreward" rather than "foreword") which is tripping up CI. I'm cautious to change it because I'm not certain it won't mess up the currently open review suggestions, so just flagging it here so that it can be cleaned up before merge.

aleesteele commented:
Sorry @da5nsy - accidentally requested your review again! Thanks for flagging the misspelling 😅 , I'll make sure to investigate this week.

aleesteele commented:
**23 April 2024**: I am adding this timeline, first drafted for governance-related work in 2022 and 2023 (formatting is a little distorted, as this was copied over from another document!). This could be added here or separately in a timeline page, perhaps in the foreword. After discussing with @malvikasharan at a coworking this week, we saw that this this is something that would be good to integrate either in this foreword section here, or in the Afterward.

I'll make sure to draft an issue with a larger structure or draft to make this more clear – for now, adding this as a comment!

| Year | Key Events |
| ----- | ---------------|
2019 | Github organisation createdGitter channel createdAll Contributors bot is added to Github repositoryCommunity governance with a focus on launching the book: https://github.com/alan-turing-institute/the-turing-way/blob/main/GOVERNANCE.md Governance discussions - early issue (issue #16)Guide for Reproducible Research CreatedTwo Book Dash Hosted in Manchester and LondonFirst set of Scriberia illustrations releasedBook extension sprint took place over two days with core team and invited external advisorsNumber of people in core team: 10 (including 1 project lead and 9 developers)Total contributors: 67Year review: https://github.com/alan-turing-institute/the-turing-way/blob/main/communications/newsletters/newsletter_09_Dec2019.md#what-an-incredible-year
2020 | Slack Workspace created (Gitter use was slowed down)Twitter page createdWelcome bot is added to Github repositoryFour more Guides were addedGovernance focusCommunity Handbook is createdAcknowledgement process establishedStyle Guide extensionContributors Record in the bookScaled chapters and resources in the book through workshops, Book Dash and Collaboration CafésGSoD intern on enhancing AccessibilityTranslation and localisation work startedPartnership formalised with organisationseScience NetherlandsTU DelftFAIR COOKBOOKNumber of people in core team: 7 (including 1 project lead, 1 CM, 2 part time RSE members, 1 GSoD intern, 2 from partnering projects)Total contributors: 235Year review: https://github.com/alan-turing-institute/the-turing-way/blob/main/communications/newsletters/newsletter_20_Dec2020.md#2020-year-in-review
2021 | Governance Taskforce idea developed & documented on Github (issue #2035)Proposal for governance extension developed (issue #2036 and HackMD)TTW HackMD workspace was createdBook Dash Committee establishedFireside Chat Series LaunchedTraining materials developed with Institutional partnership: Turing-CrickPython package for enhancing features in TTW book developed by the Turing REGAwarded HiddenREF - Highly commended for practicesNumber of people in core team: 15 (including 1 project lead, 1 CM, 5 part-time RSE members, 8 Book Dash Committee Members)Total contributors: 308Year review: https://github.com/alan-turing-institute/the-turing-way/blob/main/communications/newsletters/newsletter_30_Dec2021.md#2021-year-in-review
2022 | Slack channel made for core teamGovernance project incubated during OLS-5 program: “Exploring ‘Governance Models’ for Open Science community projects as per their maturity stage (google drive)Mastodon page createdTranslation & Localisation Team formalised3 teams & working groups incubatedReviewers & EditorsTrainers & MentorsInfrastructure MaintenanceInitial onboarding documents for core team created (HackMD)Collaboration initiated with2i2c (existing collab with JupyterHub), VU AmsterdamEnvironmental Data Science BookAwarded OpenUK - Belonging categoryNumber of people in core team: 25Total contributors: 420Year review: https://github.com/alan-turing-institute/the-turing-way/blob/main/communications/newsletters/newsletter_38_Dec2022.md#2022-year-in-review
2023 | Welcome page created (link)Practitioners Hub launchedONS, GE, BAS, ESC and DC onboarded1 working group being incubatedAccessibilityCollaboration initiated withOpenUK, FORRT, MetaDocenciaNumber of people in core team: 32 (as of 07.07.2023)

bsipocz commented:
Also, cleaned up (rebased this) in the hope of getting a passing CI.

da5nsy commented:
How do you think this should mesh with #3471 and #2033 @malvikasharan and @aleesteele?
